plugins/plugin-bitbucket/codenvy-plugin-bitbucket-ext-bitbucket-server/src/main/java/org/eclipse/che/ide/ext/bitbucket/server/BitbucketConnection.java

```diff
@@ -104,6 +104,27 @@ BitbucketPullRequest openPullRequest(String owner, String repositorySlug, Bitbuc
                                                                                                                        BitbucketException;
 
     /**
+     * Update a pull request in the Bitbucket repository.
+     *
+     * @param owner
+     *         the repository owner
+     * @param repositorySlug
+     *         the repository name
```

Comment:
As far as I understand repository name and repository slug are different things https://github.com/codenvy/codenvy/pull/2000/files#diff-66c3fbda6d05a3029ff6c6e25921e59cR27